terra-jupyter-hail/Dockerfile

```diff
@@ -1,30 +1,18 @@
 FROM us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:0.0.1
 USER root
+ENV PIP_USER=false
```

Comment:
I'm assuming we do this because hail needs to be `pip install`ed at the system level, not user level correct?